sway-fmt-v2/src/utils/helpers.rs

```diff
@@ -0,0 +1,37 @@
+use crate::Formatter;
+use sway_parse::Ty;
+use sway_types::Spanned;
+
+/// Handling of a ty, currently this does not apply any formatting.
+/// It simply pushes the string version of the ty.
+pub fn handle_ty(ty: &Ty, push_to: &mut String) {
```

Comment:
since this function has more to do with changing the type to `String`, you may want to move it to `sway_parse::Ty` as a method on `Ty` and give it a more clear name like `to_string()`

Reply:
We can continue to use `to_string` of` ty` and add this kind of routine later when we need it.